ghproxy/ghcache/ghcache.go

```diff
@@ -85,6 +85,39 @@ const (
 	apiV4 = "v4"
 )
 
+// RequestThrottlingTimes keeps the information about throttling times per API and request methods
+type RequestThrottlingTimes struct {
+	// ThrottlingTime is applied for all non-GET request methods for apiV3 and apiV4
+	throttlingTime uint
+	// ThrottlingTimeV4 if different then 0, it's applied for non-GET request methods for apiV4, instead of ThrottlingTime
+	throttlingTimeV4 uint
+	// ThrottlingTimeForGET is applied for all GET request methods for apiV3 and apiV4
+	throttlingTimeForGET uint
+}
+
+func (rtt *RequestThrottlingTimes) isEnabled() bool {
+	if rtt.throttlingTime > 0 && rtt.throttlingTimeForGET > 0 {
+		return true
+	}
+	return false
```

Comment:
Maybe just `return rtt.throttlingTime > 0 && rtt.throttlingTimeForGET > 0`?

Reply:
Yes, that was stupid. Corrected.